app/assets/stylesheets/css3/_font-face.scss

```diff
@@ -1,3 +1,26 @@
+@charset "UTF-8";
+
+/// Generates an @font-face declaration. Accepts arugments for weight, style, usage with the Rails Asset Pipeline and file formats.
+///
+/// @param {String} $font-family
+/// @param {String} $file-path
+/// @param {String} $weight [normal]
+/// @param {String} $asset-pipeline [$asset-pipeline]
+/// `$asset-pipeline` is set to `false` by default. You can pass in `true` to use the Rails Asset Pipeline (place the fonts in `app/assets/fonts/').
+/// @param {List} $file-formats [$global-font-file-formats]
+/// `$global-font-file-formats` is set to `null` by default. You can instead pass a list of file formats to support. E.g. `eot woff2 woff ttf svg`.
```

Comment:
Rather than `You can instead…`, maybe word it as something like `Pass a list of the file formats for the font…`? They will definitely _need_ to pass something in, and “instead” makes it seem optional.